test/module/irohad/torii/processor/consensus_status_processor_test.cpp

```diff
@@ -0,0 +1,284 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gmock/gmock.h>
```

Comment:
Either gmock or gtest is redundant. One includes another

Reply:
Seems they are different, but is there a proof of it?

Reply:
And anyway, there is an issue for includes. In C++ includes are transitive but better to emphase the dependencies explicitly.